src/plugins/intel_gna/tests/functional/fp32_inference_tests/fp32_inference_tests.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <vector>
+
+#include "openvino/openvino.hpp"
+#include "openvino/runtime/intel_gna/properties.hpp"
+#include "test_models.hpp"
+
+void run_test(std::shared_ptr<ov::Model> model,
```

Comment:
GFYI, we have also base test class test with implemented logic how to compare inference results on the particular models (SubgraphBaseTest), but I don't mind to use this implementation